### ISSUE TITLE
Add cross validation techniques 

### ISSUE THREAD

**Issue by KaviRana** [Rakesh9100/ML-Project-Drug-Review-Dataset#32]
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

I would like to propose adding cross-validation techniques to the drug review project. Currently, the project includes algorithms such as linear regression, logistic regression, perceptron, and decision tree. By incorporating cross-validation, we can obtain more robust evaluations of these algorithms and improve the model's performance.

I suggest using scikit-learn's cross_val_score function and KFold for the cross-validation strategy. This example can serve as a starting point for implementing cross-validation with other algorithms as well.

By incorporating cross-validation, we can better assess the performance of the models, estimate their generalization capabilities, and potentially fine-tune hyperparameters to enhance their accuracy. This addition will make the project more reliable and comprehensive.

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/ML-Project-Drug-Review-Dataset/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

**Comment by KaviRana** [comment]
@Rakesh9100 I am a GSSOC'23 contributor, please assign this issue to me.

**Comment by Rakesh9100** [comment]
Okay proceed @Rakesh9100 Assigned

**Comment by Rakesh9100** [comment]
Okay proceed @KaviRana Assigned
